dali/daliadmin/daliadmin.cpp

```diff
@@ -590,7 +590,7 @@ static void setdfspartattr(const char *lname, unsigned partNum, const char *attr
         throw MakeStringException(0, "External file not supported");
     if (lfn.isForeign()) 
         throw MakeStringException(0, "Foreign file not supported");
-    Owned<IDistributedFile> file = queryDistributedFileDirectory().lookup(lname, userDesc);
+    Owned<IDistributedFile> file = queryDistributedFileDirectory().lookup(lname, userDesc, false, false, false, nullptr, true);
     if (nullptr == file.get())
```

Comment:
normally just:
```
if (!file)
```

(as is the case elsewhere in this file)

Reply:
Unrelated change but I'll go ahead and change.